components/lcms2/Makefile

```diff
@@ -60,3 +65,7 @@ BUILD_PKG_DEPENDENCIES = $(BUILD_TOOLS)
 
 include ../../make-rules/depend.mk
```

Comment:
Style notes: Please, remove BUILD_PKG_DEPENDENCIES and include depend.mk. Also  move it to comonents/library/ and change  'include ../../make-rules/prep.mk' and similar with 'include $(WS_MAKE_RULES)/prep.mk